modules/ivc_champva/spec/jobs/missing_form_status_job_spec.rb

```diff
@@ -3,7 +3,8 @@
 require 'rails_helper'
 
 RSpec.describe 'IvcChampva::MissingFormStatusJob', type: :job do
-  let!(:forms) { create_list(:ivc_champva_form, 3, pega_status: nil) }
+  let!(:one_week_ago) { Time.now.utc - 60 * 60 * 24 * 7 }
+  let!(:forms) { create_list(:ivc_champva_form, 3, pega_status: nil, created_at: one_week_ago) }
```

Comment:
Before each test, create 3 dummy form submissions and set their `created_at` timestamp to 1 week ago.